cocos/platform/openharmony/OpenHarmonyPlatform.cpp

```diff
@@ -208,15 +208,24 @@ void OpenHarmonyPlatform::onCreateNative(napi_env env, uv_loop_t* loop) {
 void OpenHarmonyPlatform::onShowNative() {
     LOGD("OpenHarmonyPlatform::onShowNative");
     EventDispatcher::dispatchOnResumeEvent();
+    if (_timerInited) {
+        uv_timer_start(&_timerHandle, &OpenHarmonyPlatform::timerCb, 0, 1);
+    }
 }
 
 void OpenHarmonyPlatform::onHideNative() {
     LOGD("OpenHarmonyPlatform::onHideNative");
     EventDispatcher::dispatchOnPauseEvent();
+    if (_timerInited) {
+        uv_timer_stop(&_timerHandle);
+    }
 }
 
 void OpenHarmonyPlatform::onDestroyNative() {
     LOGD("OpenHarmonyPlatform::onDestroyNative");
+     if (_timerInited) {
```

Comment:
Can we use _timerHandle!=null?

Reply:
_timerHandle cannot be compared with null